### DISCUSSION TITLE
Game stops tracking tablet movement for a moment

### DISCUSSION THREAD

**Discussion by AlejandroSuero** [ppy/osu#29782] [Q&A]
### Type

Performance

### Bug description

Sometimes when playing a song, the cursor movement from the tablet (Wacom Intuos PT M), with the settings for the area set from within **osu!**'s settings, stops tracking the movement. This has happened with the version before the latest one too, so I don't think it's specific to the new changes.

Even after watching replays, you can still see how it stops and comes back again.

- OS: MacOS 14.6.1 (23G93)

### Screenshots or videos

- Video from replay of https://osu.ppy.sh/beatmapsets/2068679#osu/4356056:

https://github.com/user-attachments/assets/d0df81d8-eed0-4342-b305-d0d267a10aa1

### Version

2024.906.2

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16920708/compressed-logs.zip)

**Comment by MEMORIEmusic** [comment]
I can reproduce as well. on macOS Sequoia RC. It happens during the gameplay as well. I've lost several S and SS because of this
